### ISSUE TITLE
oelint.vars.pkgspecific.RRECOMMENDS FP

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#153]
meta/recipes-core/packagegroups/packagegroup-base.bb produces false positive for oelint.vars.pkgspecific.RRECOMMENDS at 
```
RRECOMMENDS_packagegroup-base 
```

**Comment by priv-kweihmann** [comment]
That's okay, as the setting is hidden behind some inline functions, which are *NOT* parsed at the moment - so it is expected to fail
